superset-frontend/src/SqlLab/components/SqlEditor/index.jsx

```diff
@@ -614,214 +579,145 @@ class SqlEditor extends React.PureComponent {
-          >
-            {adjustedWidth => (
-              <StyledSidebar
-                className={`schemaPane ${leftBarStateClass}`}
-                width={adjustedWidth}
-                hide={this.props.hideLeftBar}
+          {adjustedWidth => (
+            <StyledSidebar
+              className={`schemaPane ${leftBarStateClass}`}
+              width={adjustedWidth}
+              hide={hideLeftBar}
+            >
+              <SqlEditorLeftBar
+                database={database}
+                queryEditor={queryEditor}
+                tables={tables}
+                actions={actions}
+                setEmptyState={bool => setShowEmptyState(bool)}
+              />
+            </StyledSidebar>
+          )}
+        </ResizableSidebar>
+      </CSSTransition>
+      {showEmptyState ? (
+        <EmptyStateBig
+          image="vector.svg"
+          title={t('Select a database to write a query')}
+          description={t(
+            'Choose one of the available databases from the panel on the left.',
+          )}
+        />
+      ) : (
+        queryPane()
+      )}
+      <Modal
+        visible={showCreateAsModal}
+        title={t(createViewModalTitle)}
+        onHide={() => setShowCreateAsModal(false)}
+        footer={
+          <>
+            <Button onClick={() => setShowCreateAsModal(false)}>Cancel</Button>
+            {createAs === CtasEnum.TABLE && (
+              <Button
+                buttonStyle="primary"
+                disabled={ctas.length === 0}
+                onClick={createTableAs}
               >
-                <SqlEditorLeftBar
-                  database={this.props.database}
-                  queryEditor={this.props.queryEditor}
-                  tables={this.props.tables}
-                  actions={this.props.actions}
-                  setEmptyState={this.setEmptyState}
-                />
-              </StyledSidebar>
-            )}
-          </ResizableSidebar>
-        </CSSTransition>
-        {this.state.showEmptyState ? (
-          <EmptyStateBig
-            image="vector.svg"
-            title={t('Select a database to write a query')}
-            description={t(
-              'Choose one of the available databases from the panel on the left.',
+                Create
+              </Button>
             )}
-          />
-        ) : (
-          this.queryPane()
-        )}
-        <StyledModal
-          visible={this.state.showCreateAsModal}
-          title={t(createViewModalTitle)}
-          onHide={() => {
-            this.setState({ showCreateAsModal: false });
-          }}
-          footer={
-            <>
+            {createAs === CtasEnum.VIEW && (
               <Button
-                onClick={() => this.setState({ showCreateAsModal: false })}
+                buttonStyle="primary"
+                disabled={ctas.length === 0}
+                onClick={createViewAs}
               >
-                Cancel
+                Create
```

Comment:
```suggestion
                {t('Create')}
```
This string needs to be translated